docs/releases/dev.rst

```diff
@@ -35,7 +35,8 @@ Details of changes
 - :setting:`POOTLE_TM_SERVER` now accepts a ``MIN_SIMILARITY`` parameter, to
   filter out results which might be irrelevant. To learn more, check the
   documenation on :setting:`MIN_SIMILARITY <POOTLE_TM_SERVER-MIN_SIMILARITY>`.
-
+- :setting:`POOTLE_EXPORT_VIEW_LIMIT` was added, with a default value of 10000.
+  This restricts how many units are show in export views.
```

Comment:
Typo: *are shown*.

As a nit, note the rest of the items in the release notes are written in present perfect, as opposed to past simple. Therefore it'd be preferable to have all items under the same tense.